pkg/kubelet/kubelet.go

```diff
@@ -1250,6 +1252,13 @@ type Kubelet struct {
 
 	// OpenTelemetry Tracer
 	tracer trace.Tracer
+
+	// Maximum number of checkpoints per container
+	maxContainerCheckpoints int32
+
+	// updateMaxContainerCheckpoints is a lock on updating the maximum number of
+	// checkpoints per container metadata file.
+	updateMaxContainerCheckpoints sync.Mutex
```

Comment:
I'd suggest to name it `maxContainerCheckpointsLock` or `maxContainerCheckpointsMutex`

Reply:
changed